staging/src/k8s.io/client-go/util/flowcontrol/backoff.go

```diff
@@ -67,15 +68,45 @@ func (p *Backoff) Get(id string) time.Duration {
 	return delay
 }
 
+// GetWithNbRetry the current backoff Duration and the number of previous retry
+func (p *Backoff) GetWithNbRetry(id string) (time.Duration, int32) {
+	p.Lock()
```

Comment:
With this new method the other `Get` should be:
```go
func (p *Backoff) Get(id string) time.Duration {
    delay, _ := p.GetWithRetryNumber(id)
    return delay
}
```
We don't want copy & paste here.

Reply:
right, usually not me either.